### ISSUE TITLE
State is not displayed for soundtouch device

### ISSUE THREAD

**Issue by Krocko** [kalkih/mini-media-player#245]
Hello.
Is it possible to display the state for soundtouch devices?

This is from an entity card:
<img width="359" alt="Bildschirmfoto 2020-03-15 um 12 39 17" src="https://user-images.githubusercontent.com/29394115/76700706-833d3c00-66ba-11ea-96d8-82d1108e36ee.png">

And this is for the custom:mini-media_player:
<img width="356" alt="Bildschirmfoto 2020-03-15 um 12 38 40" src="https://user-images.githubusercontent.com/29394115/76700718-9cde8380-66ba-11ea-967c-94bcd7b8a8b7.png">

Would also be nice for echo devices.

Entity card:
<img width="354" alt="Bildschirmfoto 2020-03-15 um 12 44 53" src="https://user-images.githubusercontent.com/29394115/76700745-d616f380-66ba-11ea-8428-060d0b7d91ea.png">

custom:mini-media_player:
<img width="348" alt="Bildschirmfoto 2020-03-15 um 12 45 54" src="https://user-images.githubusercontent.com/29394115/76700758-f181fe80-66ba-11ea-846b-6c613df42ad4.png">

And is it possible to get the volume slider for the echo device displayed?

**Comment by kalkih** [comment]
Hey,

I guess the default entity card displays the state when there's no media metadata available.
Could possibly add the sam behaviour to this card.

Volume slider should work as long as the component supports it, you didn't hide it in the card config by mistake? Otherwise try the [Alexa media player custom component](https://github.com/custom-components/alexa_media_player).

**Comment by Krocko** [comment]
Thank you for your answer.
Would be nice if you could implement this.

I have the Alexa custom component installed. 
I don‘t hide the volume slider. 
The volume is displayed if media is playing. 
But I mean, if it is possible to display the tts volume slider if tts is enabled.

**Comment by kalkih** [comment]
> I have the Alexa custom component installed.
> I don‘t hide the volume slider.
> The volume is displayed if media is playing.
> But I mean, if it is possible to display the tts volume slider if tts is enabled.

Not really, the volume slider will not be displayed when the device is off.
You could always create a universal media player and override the state to make the device report as always on, that way you should be able to adjust the volume before sending tts (if the alexa integration allow this, I know chromecasts does).
